CHANGELOG.md

```diff
@@ -13,8 +13,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add styling guide documentation for customizing SDK and UI component CSS.
 - Extend and enabled style customizing capabilities on the SDK components.
 ### Changed
-
+- Update meeting manager to use new event controller API
+- Update node module to v3 beta
```

Comment:
nit: need to add period at the end of line
```
Update meeting manager to use new event controller API
Update node module to v3 beta
```
to
``````
Update meeting manager to use new event controller API.
Update node module to v3 beta.
```

Reply:
Fixed